docs/topics/pagination.txt

```diff
@@ -2,15 +2,68 @@
 Pagination
 ==========
 
+Django provides high-level and low-level ways to help you manage paginated data
+-- that is, data that's split across several pages, with "Previous/Next" links.
+
+Paginating a `ListView`
+=======================
+
+:class:`django.views.generic.list.ListView` provides a built-in way to
+paginate the displayed list. You can use it by adding to ``paginate_by``
+attribute to your view class, like so::
+
+    from django.views.generic import ListView
+    from books.models import Publisher
+
+    class PublisherList(ListView):
+        paginate_by = 2
+        model = Publisher
+
+Assuming you were able to view the full list of publishers before this change,
+the example above already works! The only thing your users will be missing is
+a way to get to the next or previous page comfortably.
```

Comment:
"comfortably" seems like an odd word. I'd be tempted to leave it out, and just say something like "... a way to navigate to the next or previous page"